src/main.py

```diff
@@ -66,6 +78,7 @@ def handleGET(path):
 
 
 def handleHEAD(path):
+    '''Generates an OK response with only the file information'''
```

Comment:
I'd say "containing only the headers", since there are not only headers related to the file.